blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Enhancing Your Operating System's Security with OS Security Patches in AKS"
+description: "Learn the what, the whys, and the hows of configuring OS Security Patch Auto upgrade channel. The article also provides a compelling case for OS Security Patch citing performance, better security, minimal disruption to workloads as chief gains."
+date: 2025-04-22
+authors:
+- Kaarthikeyan Subramanian
+- Ye Wang
+categories: 
+- Security
+- Workload SLO
+- Upgrades
+---
+
+
+## Why do we need security patches at the Node Operating System level?
+
+Operating System (OS) security patches are crucial for maintaining the integrity and security of your systems. These patches address vulnerabilities that could be exploited by malicious actors, ensuring that your system remains protected against the latest threats. Regularly applying OS security patches helps to safeguard sensitive data, maintain compliance with security standards, and prevent potential disruptions caused by security breaches . At Azure Kubernetes service we are constantly working hard and innovating to guarantee maximum security for your Node operating systems. 
+Presently there are two ways to apply security patches at the OS level automatically with AKS. Read more [OS Security Patch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli)
+
+**Auto Node Image Channel** - AKS updates nodes weekly with a newly patched VHD for security and bug fixes. This update, which follows maintenance windows and surge settings, incurs no extra VHD cost. Linux unattended upgrades are disabled by default when using this channel. Node image upgrades are supported as long as the cluster's Kubernetes minor version is in support. These AKS-tested node images are fully managed and applied with safe deployment practices.
+**OS Security Patch Channel** - Several customers may need only the security packages for their OS without additional bug fixes and updates. Customers previously relied on nightly updates and reboot mechanisms like KURED for security updates. Past outages from untested packages highlighted the need for a managed security patch-only channel at the node level.  With OS Security Patch channel you get precisely that a fully managed attended Node OS security only channel! This automates manual processes, eliminating the need for tools like KURED. The Security-Patch channel reimages nodes only when necessary and provides live patch updates with zero disruption, respecting planned maintenance windows.
+
+## OS Security Patch channel how it fares against Node Image Channel
```

Comment:
So is there any case where customer still want to have NodeImage channel (I guess there is scenario where they want to be disrupted less frequently?), suggesting adding a sentence to explain

Reply:
yes they can consider NodeImage if they are interested in latest VHD esp with the bug fixes and features outside security patch .